frontend/src/components/navigation/Navbar.jsx

```diff
@@ -1,40 +1,55 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-import './navbar.css';
+import '../../styles/components/navbar.css';
+import '../../styles/base/typography.css';
+import '../../styles/base/variables.css';
```

Comment:
These 3 style imports can be removed from here if we add `@import url('./components/footer.css');` to` style.css` under the components section